scripts/obtain/dotnet-install.sh

```diff
@@ -648,6 +655,8 @@ do
             echo "  --no-path, -NoPath             Do not set PATH for the current process."
             echo "  --verbose,-Verbose             Display diagnostics information."
             echo "  --azure-feed,-AzureFeed        Azure feed location. Defaults to $azure_feed"
+            echo "  --linux-portable               Installs the Linux portable shared runtime instead of the shared runtime for OS."
+			echo "      -LinuxPortable"
```

Comment:
Followed up with Eric offline. This is fine since we are supporting multiple param names. This follows the convention with the other params.